scripts/sct_register_multimodal.py

```diff
@@ -233,13 +233,17 @@ def addStep(self, stepParam):
         # this function checks if the step is already present. If it is present, it must update it. If it is not, it must add it.
         param_reg = Paramreg()
         param_reg.update(stepParam)
-        if param_reg.step != 0:
+        # step is mandatory
```

Comment:
I might be missing something here. I don't see in the code the step conversion from string to int.